.github/workflows/main.yml

```diff
@@ -215,16 +220,21 @@ jobs:
           # Try deleting any previous build, but not other zip files that we may need later.
           git clean -fd -e 'build-*.zip'
           rm -rf build "build-${{ matrix.targetPlatform }}.zip" || true
-      - uses: game-ci/unity-builder@7afabe74da5227a9ecd537f5260b398258c3eb3b
+      - name: Restore NuGet packages
+        run: /home/see/.dotnet/tools/nugetforunity restore .
+      - uses: game-ci/unity-builder@v4
         timeout-minutes: 45
         name: Build project
         id: build
         env:
           UNITY_LICENSE: ${{ secrets.UNITY_LICENSE }}
+          UNITY_EMAIL: ${{ secrets.UNITY_EMAIL }}
+          UNITY_PASSWORD: ${{ secrets.UNITY_PASSWORD }}
```

Comment:
When using these secrets in scripts, make sure to pipe all outputs to `dev/null`.

Reply:
Yes, that's usually a good idea, but in this case I need the output of the `game-ci` scripts to be sent to stdout, as GitHub Actions would not catch [errors or warnings](https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions) otherwise. Nevertheless, this should not be an issue because:
* The `game-ci` scripts take care never to output any of these secrets, and more importantly
* GitHub Actions searches the output for the presence of any secrets and censors them.